### PR TITLE
Changed /kits to only give bow/arrows if kit is Assassin or Ranger

### PR DESCRIPTION
**Pull request by StudentAlleg**

### CODE REVIEW

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/commands/menu/KitButton.java [hunk 1]
```diff
@@ -29,8 +29,19 @@ public void onClick(Player player, Gamer gamer, ClickType clickType) {
...
+        if((role == Role.ASSASSIN) || (role == Role.RANGER)) {
+            player.getInventory().addItem(new ItemStack(Material.BOW));
+
+            int numArrows;
+
+            if (role == Role.RANGER) {
+                numArrows = 64;
+            }
+            else {
+                numArrows = 16;
```
Comment (Mykindos) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/commands/menu/KitButton.java:41]: maybe 32 arrows?

numArrows = (role == Role.RANGER ? 64 : 32);

**Review by Mykindos — APPROVED** [review]

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/commands/menu/KitButton.java (hunk 1 above)
Comment (StudentAlleg) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/commands/menu/KitButton.java:41]: /shrug on balancing

Comment (Mykindos) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/commands/menu/KitButton.java]: The command is really only used for testing at this stage

Comment (StudentAlleg) [reply champions/src/main/java/me/mykindos/betterpvp/champions/champions/commands/menu/KitButton.java]: fair enough

**Review by Mykindos — APPROVED** [review]

**Pull request merged**